site/ko/tutorials/keras/basic_classification.ipynb

```diff
@@ -49,6 +49,8 @@
     "#\n",
     "# Copyright (c) 2017 François Chollet\n",
     "#\n",
+    "# IGNORE_COPYRIGHT: cleared by OSS licensing\n",
```

Comment:
I think you can ignore this for now. We don't like the way this looks and are working with the OSS team to make this better